### ISSUE TITLE
Train with dedicated test set

### ISSUE THREAD

**Issue by tmbo** [RasaHQ/rasa_nlu#1982]
Allow a user to specify a separate story data set when training that is used for the evaluation when training. This should replace the automatic splitting used by keras to determin the validation dataset. If no test set is passed, the current behaviour should be used (splitting the training data set and evaluate on parts of it).

**Comment by amn41** [comment]
covered by https://rasa.com/docs/core/evaluation/#comparing-policies
